bin/pasta_analyse.py

```diff
@@ -295,12 +307,21 @@ def fill_result(hashes, tag):
 
             type = EvaluationType.PatchStack
 
-        log.info('Starting evaluation')
-        evaluation_result = evaluate_commit_list(repo, config.thresholds,
-                                                 mbox, type,
-                                                 representatives, candidates,
-                                                 parallelise=True, verbose=True,
-                                                 cpu_factor=args.cpu_factor)
+        if args.differential:
+            representatives = representatives | new_patches
+            log.info('Starting differential evaluation of %u new patches' % len(new_patches))
+            evaluation_result = evaluate_commit_list(repo, config.thresholds,
+                                                     mbox, type,
+                                                     representatives, new_patches,
+                                                     parallelise=True, verbose=True,
+                                                     cpu_factor=args.cpu_factor)
+        else:
+            log.info('Starting evaluation')
+            evaluation_result = evaluate_commit_list(repo, config.thresholds,
+                                                     mbox, type,
+                                                     representatives, candidates,
+                                                     parallelise=True, verbose=True,
+                                                     cpu_factor=args.cpu_factor)
```

Comment:
Both calls of evaluate_commit_list are almost the same, besides we have new_patches instead of candidates in the differential case. Can't we just say:

```
if differential:
    candidates = new_patches
```